app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,35 @@
+<nav class="navbar navbar-expand-lg bg-light mb-4">
+  <div class="container">
+    <%= link_to "Photogram", root_path, class: "navbar-brand" %>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+      <ul class="navbar-nav">
+
+        <% if user_signed_in? %>
+          <li class="nav-item">
+            <%= link_to "Edit #{current_user.username}", edit_user_registration_path, class: "nav-link" %>
+          </li>
+          <li class="nav-item">
+            <%= link_to "Sign out", destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" %>
+          </li>
+        <% else %>
+            <li class="nav-item">
+              <%= link_to "Sign in", new_user_session_path, class: "nav-link" %>
+            </li>
+            <li class="nav-item">
+              <%= link_to "Sign up", new_user_registration_path, class: "nav-link" %>
+            </li>
+        <% end %>  
+
+          <%# <ul class="dropdown-menu">
```

Comment:
Its good practice to delete commented out code that you wont be using anymore just to keep things clean :)